tools/matc/src/matc/ParametersProcessor.cpp

```diff
@@ -387,6 +391,30 @@ bool ParametersProcessor::processBlending(filamat::MaterialBuilder& builder,
     return true;
 }
 
+bool ParametersProcessor::processPostLightingBlending(filamat::MaterialBuilder& builder,
+        const JsonishValue& value) {
+    auto jsonString = value.toJsonString();
+    if (!isStringValidEnum(mStringToBlendingMode, jsonString->getString())) {
+        std::cerr << PARAM_KEY_BLENDING << ": value is not a valid BlendMode." << std::endl;
+        return false;
+    }
+
+    filament::BlendingMode postLightingBlending =
+            stringToEnum(mStringToBlendingMode, jsonString->getString());
+    switch (postLightingBlending) {
+        case filament::BlendingMode::FADE:
+        case filament::BlendingMode::MASKED:
+            std::cerr << PARAM_KEY_BLENDING <<
+                    ": value is not a valid post-lighting BlendMode."
+                    " Only OPAQUE, TRANSPARENT and FADE are supported." << std::endl;
```

Comment:
Shouldn't it be "OPAQUE, TRANSPARENT, and ADD"?

Reply:
Indeed